docs/yul.rst

```diff
@@ -899,6 +899,7 @@ the ``dup`` and ``swap`` instructions as well as ``jump`` instructions, labels a
 | revert(p, s)            | `-` | B | end execution, revert state changes, return data mem[p...(p+s)) |
 +-------------------------+-----+---+-----------------------------------------------------------------+
 | selfdestruct(a)         | `-` | F | end execution, destroy current contract and send funds to a     |
+|                         |     |   | (see warning below)                                             |
```

Comment:
I'd just say right away it's deprecated.

```suggestion
|                         |     |   | (deprecated)                                                    |
```

Reply:
I'm fine with that as well - the reason I used `(see warning below)` is mainly because it follows the same format as `difficulty` and `prevrandao`, although these are not deprecated, so you may have a point.